### DISCUSSION TITLE
Blog Post: Deep Dive into Hello World

### DISCUSSION THREAD

**Discussion by altugbakan** [stellar/sorobanathon#74] [General]
Hello again!

On this blog post, I explained the Hello World contract in as much detail as possible. I wrote about how contracts work, how are they built, and how tests are done. Please check out my blogpost [here](https://dev.to/altug/soroban-quest-hello-world-1n3l) on dev.to.

Please share your thoughts with me so I can improve my content!

**Comment by achallagundla** [comment]
Heya @altugbakan  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
